test/unit_test/runtime/magnitude_test.cpp

```diff
@@ -133,6 +133,11 @@ TEST_CASE("make_ratio performs prime factorization correctly")
     REQUIRE(r.exp == 2);
     CHECK(as_magnitude<r>() == as_magnitude<300>());
   }
+
+  SECTION("Can support exp which would be large enough to cause overflow")
```

Comment:
This section header can be improved.
Cause overflow how? If it's supported, then it's because there's no overflow, right?

Reply:
Good catch.  Does this new version (plus explanatory comment) read better to you?

Reply:
It's the best it can be.